lms/templates/courseware/courseware.html

```diff
@@ -29,6 +29,10 @@
 <%block name="headextra">
 <%static:css group='style-course-vendor'/>
 <%static:css group='style-course'/>
+## Utility: Notes
```

Comment:
What do you think about moving `lms/static/sass/course/modules/_student-notes.scss` in this group?